frontend/src/packages/auth/auth-api.package.ts

```diff
@@ -3,6 +3,8 @@ import { HttpApi } from '~/libs/packages/api/api.js';
 import { type IHttp } from '~/libs/packages/http/http.js';
 import { type IStorage } from '~/libs/packages/storage/storage.js';
 import {
+  type UserAuthRequestDto,
+  type UserAuthResponseDto,
   type UserSignUpRequestDto,
   type UserSignUpResponseDto,
 } from '~/packages/users/users.js';
```

Comment:
```suggestion
import type { 
UserAuthRequestDto, 
UserAuthResponseDto,  
UserSignUpRequestDto, 
UserSignUpResponseDto, 
} from '~/packages/users/users.js';
```

Reply:
Linter said that I should prefer version with type near each import. 
`Avoid import/export type { Type } from './module'. Prefer import/export { type Type } from './module'.eslint[no-restricted-syntax](https://eslint.org/docs/latest/rules/no-restricted-syntax)`